### PR TITLE
chore: update Input, Input Box styles

### PR DESCRIPTION
**Pull request by gloriliale**
# In this PR
Update styles in Input and Input Box to reflect the new theme. Also updates docs to reflect implementation changes.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-core/components/input-box/src/Prefix.tsx [hunk 1]
```diff
@@ -19,13 +19,13 @@ export interface PrefixProps {
 
 const Prefix = React.forwardRef<HTMLDivElement, PrefixProps>(
   ({children, disabled, element = 'PREFIX', variant}, ref) => {
-    let backgroundColor = 'colorBackground' as BackgroundColor;
+    let backgroundColor = 'colorBackgroundWeak' as BackgroundColor;
     let borderColor = 'colorBorderWeaker' as BorderColor;
     if (disabled && variant === 'inverse') {
       backgroundColor = 'none';
       borderColor = 'colorBorderInverseWeaker';
     } else if (variant === 'inverse') {
```
Comment (serifluous) [inline packages/paste-core/components/input-box/src/Prefix.tsx:27]: todo: add a conditional for a disabled state to apply these styles:
- no background color
- color-border-weaker

File: packages/paste-core/components/input-box/src/Suffix.tsx [hunk 2]
```diff
@@ -19,13 +19,13 @@ export interface SuffixProps {
 
 const Suffix = React.forwardRef<HTMLDivElement, SuffixProps>(
   ({children, disabled, element = 'SUFFIX', variant}, ref) => {
-    let backgroundColor = 'colorBackground' as BackgroundColor;
+    let backgroundColor = 'colorBackgroundWeak' as BackgroundColor;
     let borderColor = 'colorBorderWeaker' as BorderColor;
     if (disabled && variant === 'inverse') {
       backgroundColor = 'none';
       borderColor = 'colorBorderInverseWeaker';
     } else if (variant === 'inverse') {
```
Comment (serifluous) [inline packages/paste-core/components/input-box/src/Suffix.tsx:27]: todo: add a conditional for a disabled state in Suffix, too

File: packages/paste-core/components/combobox/stories/Combobox.stories.tsx [hunk 3]
```diff
@@ -392,7 +392,7 @@ export const ComboboxDisabledInsertBeforeAndAfter: StoryFn = () => {
     <Combobox
       items={items}
       insertBefore={
-        <Text as="span" fontWeight="fontWeightSemibold">
+        <Text color="colorTextWeak" as="span" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/combobox/stories/Combobox.stories.tsx:395]:
```suggestion
        <Text color="colorTextWeaker" as="span" fontWeight="fontWeightSemibold">
```

File: packages/paste-core/components/combobox/stories/Combobox.stories.tsx [hunk 4]
```diff
@@ -441,7 +441,7 @@ export const ComboboxDisabledInsertBeforeAndAfterInverse: StoryFn = () => {
       <Combobox
         items={items}
         insertBefore={
-          <Text as="span" color="colorTextInverse" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
+          <Text as="span" color="colorTextInverseWeak" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/combobox/stories/Combobox.stories.tsx:444]:
```suggestion
          <Text as="span" color="colorTextInverseWeaker" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```

File: packages/paste-core/components/input/stories/input.stories.tsx [hunk 5]
```diff
@@ -458,7 +458,7 @@ export const InputDisabledInsertBeforeAndAfter = (): React.ReactNode => {
         onFocus={action('handleFocus')}
         onBlur={action('handleBlur')}
         insertBefore={
-          <Text as="span" fontWeight="fontWeightSemibold">
+          <Text color="colorTextWeak" as="span" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/input/stories/input.stories.tsx:461]:
```suggestion
          <Text color="colorTextWeaker" as="span" fontWeight="fontWeightSemibold">
```

File: packages/paste-core/components/input/stories/input.stories.tsx [hunk 6]
```diff
@@ -535,7 +535,7 @@ export const InputDisabledInsertBeforeAndAfterInverse = (): React.ReactNode => {
         onBlur={action('handleBlur')}
         variant="inverse"
         insertBefore={
-          <Text as="span" color="colorTextInverse" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
+          <Text as="span" color="colorTextInverseWeak" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/input/stories/input.stories.tsx:538]:
```suggestion
          <Text as="span" color="colorTextInverseWeaker" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```

File: packages/paste-core/components/select/stories/select.stories.tsx [hunk 7]
```diff
@@ -333,7 +333,7 @@ export const SelectDisabledInsertBeforeAndAfter = (): React.ReactNode => {
         onFocus={action('handleFocus')}
         onBlur={action('handleBlur')}
         insertBefore={
-          <Text as="span" fontWeight="fontWeightSemibold">
+          <Text color="colorTextWeak" as="span" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/select/stories/select.stories.tsx:336]:
```suggestion
          <Text color="colorTextWeaker" as="span" fontWeight="fontWeightSemibold">
```

File: packages/paste-core/components/select/stories/select.stories.tsx [hunk 8]
```diff
@@ -415,7 +415,7 @@ export const SelectDisabedInsertBeforeAndAfterInverse = (): React.ReactNode => {
         onFocus={action('handleFocus')}
         onBlur={action('handleBlur')}
         insertBefore={
-          <Text as="span" color="colorTextInverse" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
+          <Text as="span" color="colorTextInverseWeak" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/select/stories/select.stories.tsx:418]:
```suggestion
          <Text as="span" color="colorTextInverseWeaker" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```

File: packages/paste-core/components/textarea/stories/textarea.stories.tsx [hunk 9]
```diff
@@ -379,7 +379,7 @@ export const TextareaDisabledInsertBeforeAndAfter = (): React.ReactNode => {
         onFocus={action('handleFocus')}
         onBlur={action('handleBlur')}
         insertBefore={
-          <Text as="span" fontWeight="fontWeightSemibold">
+          <Text color="colorTextWeak" as="span" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/textarea/stories/textarea.stories.tsx:382]:
```suggestion
          <Text color="colorTextWeaker" as="span" fontWeight="fontWeightSemibold">
```

File: packages/paste-core/components/textarea/stories/textarea.stories.tsx [hunk 10]
```diff
@@ -447,7 +447,7 @@ export const TextareaDisabledInsertBeforeAndAfterInverse = (): React.ReactNode =
         onFocus={action('handleFocus')}
         onBlur={action('handleBlur')}
         insertBefore={
-          <Text as="span" color="colorTextInverse" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
+          <Text as="span" color="colorTextInverseWeak" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```
Comment (serifluous) [inline packages/paste-core/components/textarea/stories/textarea.stories.tsx:450]:
```suggestion
          <Text as="span" color="colorTextInverseWeaker" lineHeight="lineHeight20" fontWeight="fontWeightSemibold">
```

**Review by nkrantz — COMMENTED** [review]
This is looking good! Not related but a bug I noticed in the input docs. If you do any more changes, could you throw this in there too?

Line 193 of `input/index.mdx` should be `id="message_title"` so that the label and input are properly linked.

File: packages/paste-core/components/input/src/Input.tsx [hunk 11]
```diff
@@ -68,11 +68,11 @@ export const InputElement = React.forwardRef<HTMLInputElement, InputProps>(({ele
       variant={props.variant}
       ref={ref}
       _placeholder={{
-        color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorTextWeak',
+        color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorText',
         fontStyle: 'italic',
       }}
       _focus_placeholder={{
-        color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorTextWeak',
+        color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorText',
```
Comment (serifluous) [inline packages/paste-core/components/input/src/Input.tsx:75]: todo: sorry 🤦‍♀️ Can you swap these back to colorTextWeak?

Comment (gloriliale) [reply packages/paste-core/components/input/src/Input.tsx]: done!

File: .changeset/tricky-cups-camp.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
+---
+'@twilio-paste/input': patch
+'@twilio-paste/input-box': patch
+'@twilio-paste/core': patch
+---
+
+[Input, Input Box] Update styles to align with new Paste Twilio theme
```
Comment (shleewhite) [inline .changeset/tricky-cups-camp.md:7]: Nit (non-blocking): it doesn't look like theres any changes to the input package anymore, so you can probs just make this a patch for input-box

**Review by shleewhite — APPROVED** [review]

**Review by serifluous — APPROVED** [review]

File: .changeset/tricky-cups-camp.md (hunk 12 above)
Comment (gloriliale) [inline .changeset/tricky-cups-camp.md:7]:
```suggestion
'@twilio-paste/input-box': patch
'@twilio-paste/core': patch
---

[Input Box] Update styles to align with new Paste Twilio theme
```

Comment (gloriliale) [reply .changeset/tricky-cups-camp.md]: oh good call, fixed it up!

**Review by Niznikr — APPROVED** [review]

**Pull request merged**